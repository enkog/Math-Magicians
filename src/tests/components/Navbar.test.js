import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../../components/UI/Navbar';

it('should render an empty message when no books', () => {
    const component = renderer.create(
      <Router>
        <Navbar />
      </Router>
    ).toJSON();
    expect(component).toMatchSnapshot();
});