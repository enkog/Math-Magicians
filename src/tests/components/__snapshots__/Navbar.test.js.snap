// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an empty message when no books 1`] = `
<nav
  className="navbar"
>
  <a
    className="nav-brand"
    href="/"
  >
    Math Magicians
  </a>
  <ul>
    <li>
      <a
        aria-current="page"
        className="active-link"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
